.travis.yml

```diff
@@ -128,7 +131,7 @@ matrix:
         - (cd dex-contracts && yarn --frozen-lockfile && yarn run prepack)
       script:
         - docker-compose -f docker-compose.yml -f docker-compose.rinkeby.yml -f docker-compose.binary.yml up -d stablex
-        - ./test/e2e-tests-stablex-rinkeby.sh
+        - cargo test -p e2e rinkeby -- --nocapture
```

Comment:
What is the -p

Does this have something to do with private key?

Reply:
`-p` for `--package`, as in the crate to test. All the e2e tests are part of the `e2e` crate.

Reply:
Given that the test function itself is called `test_rinkeby` and we are calling `rinkeby`, is this cargo test doing some sort of string matching?

Reply:
Yes, it's pattern matching the string with the test names, cf. https://doc.rust-lang.org/rust-by-example/cargo/test.html